pyFAST/fastfarm/FASTFarmCaseCreation.py

```diff
@@ -343,33 +293,30 @@ def _checkInputs(self):
         
         # Set domain extents defaults if needed
         default_extent_low  = [3,6,3,3,2]
-        default_extent_high = 1.2          # total extent, half of that to each side.
+        default_extent_high = 1.2
```

Comment:
I see no reason to delete this comment.

Reply:
For floating this is not a big enough default and might be misleading for whoever is running the simulation. I prefer if we give an error than to set a default value.